### ISSUE TITLE
save method is EntrustUserTrait return void!

### ISSUE THREAD

**Issue by ahmadazimi** [Zizaco/entrust#514]
Hi there,

I've found that you never return anything  on **save** and **delete** methods in the _EntrustUserTrait_ class.
The parent class (`Illuminate\Database\Eloquent\Model`) return status of that action for save and delete method.

Is there any idea behind your decision to make this method return void?

**Comment by 19peaches** [comment]
Yeah I found this same situation on EntrustRoleTrait as well. I made local copies to get around it but indeed, something needs to get returned, I spent a couple hours chasing this down.

**Comment by ahmadazimi** [comment]
Hi there,

Is there any chance to fix this issue on any version of this plugin?
I'm still waiting for it...

**Comment by ahmadazimi** [comment]
Is this repo still alive?
